### PR TITLE
test_delete_custom_fields

### PR DESCRIPTION
**Pull request by Bharani0012**
Added New testcases for delete Custom field api

updated delete custom field functions

### CODE REVIEW

File: test/IntegrationTests/TicketService/test_delete_custom_field.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+class TestDeleteCustomFieldAPI(unittest.TestCase):
+    BASE_URL = "http://localhost:8000"
```
Comment (SaravanakumarR2018) [inline test/IntegrationTests/TicketService/test_delete_custom_field.py:14]: Use Env variable for 8000 

Use CHAT_SERVICE_PORT

Use env variables for all new files

File: test/IntegrationTests/TicketService/test_delete_custom_field.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                         f"Expected 404 Not Found, but got {response.status_code} for URL: {invalid_url}")
+
+        # Optionally, check that the response contains a message indicating the endpoint does not exist
+        response_data = response.json()
+        self.assertIn("detail", response_data, "Response should contain a 'detail' key.")
+        self.assertEqual(response_data["detail"], "Not Found", "Expected 'Not Found' message in the response.")
+
+        logger.info(f"Successfully validated 404 Not Found response for invalid endpoint: {invalid_url}")
+
+    def tearDown(self):
```
Comment (SaravanakumarR2018) [inline test/IntegrationTests/TicketService/test_delete_custom_field.py:213]: You can also delete the customer that you have created in setUp?

File: test/IntegrationTests/TicketService/test_delete_custom_field.py [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+            response = requests.post(custom_field_url, json=field_data)
+            self.assertEqual(
+                response.status_code,
+                HTTPStatus.CREATED,
+                f"Failed to add custom field {field['field_name']}"
+            )
+
+        # Step 2: Verify all custom fields are added
+        initial_fields = self.list_custom_fields()
+        self.assertEqual(len(initial_fields), len(custom_fields), "Custom fields count mismatch after addition")
```
Comment (SaravanakumarR2018) [inline test/IntegrationTests/TicketService/test_delete_custom_field.py:75]: Can we also check the names of the custom fields and its field type here and compare it with the input?

Comment (Bharani0012) [reply test/IntegrationTests/TicketService/test_delete_custom_field.py:75]: modified sir

File: test/IntegrationTests/TicketService/test_delete_custom_field.py (hunk 2 above)
Comment (Bharani0012) [reply test/IntegrationTests/TicketService/test_delete_custom_field.py:213]: sir, we have delete customer db method in database manager, but we dont have a api end point, can i create new one.

**Review by SaravanakumarR2018 — APPROVED** [review]
Approved

**Pull request merged**